sdc/tests/test_groupby.py

```diff
@@ -53,6 +54,272 @@
 
 
 class TestGroupBy(TestCase):
+
+    @skip_sdc_jit('Fails with old-pipeline from the start')
+    @sdc_limitation
+    def test_dataframe_groupby_index_name(self):
+        """SDC indexes do not have names, so index created from a named Series looses it's name."""
+        def test_impl(df):
+            return df.groupby('A').min()
+        hpat_func = self.jit(test_impl)
+
+        n = 11
+        df = pd.DataFrame({
+                    'A': [2, 1, 1, 1, 2, 2, 1, 0, 3, 1, 3],
+                    'B': np.arange(n, dtype=np.intp)
+        })
+        result = hpat_func(df)
+        result_ref = test_impl(df)
+        pd.testing.assert_frame_equal(result, result_ref)
+
+    @skip_sdc_jit('Fails with old-pipeline from the start')
+    def test_dataframe_groupby_count_by_int(self):
```

Comment:
Maybe to combine tests `test_dataframe_groupby_count_by_*` via `subTest`?

Reply:
Done!